#this message will be used for motor data infor
MotorData[] motorData
